src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsComputeQuota.ps1

```diff
@@ -153,9 +147,9 @@ function New-AzsComputeQuota {
                 }
 
                 $quotaObj = Get-TaskResult @GetTaskResult_params
-				[QuotaCustomObject]$script:result = New-QuotaCustomObject -Quota $quotaObj
+                [ComputeQuotaObject]$script:result = ConvertTo-ComputeQuota -Quota $quotaObj
```

Comment:
You don't need to save to variable.  Just have this single line at the end of your function

```
ConvertTo-ComputeQuota -Quota (Get-TaskResult @GetTaskResult_params)
```